src/Serilog.Sinks.MongoDB/LoggerConfigurationMongoDBExtensions.cs

```diff
@@ -122,11 +121,15 @@ public static class LoggerConfigurationMongoDBExtensions
             if (loggerConfiguration == null) throw new ArgumentNullException("loggerConfiguration");
             if (databaseUrl == null) throw new ArgumentNullException("databaseUrl");
 
-            var optionsBuilder = CollectionOptions.SetCapped(true).SetMaxSize(cappedMaxSizeMb * 1024 * 1024);
+            var options = new CreateCollectionOptions
+            {
+                Capped = true,
+                MaxSize = cappedMaxSizeMb * 1024 * 1024
+            };
 
```

Comment:
Aah - guessing by the nature of these, the API has changed too. Makes the changes above more clear. Thanks!